tests/tests_pytorch/trainer/connectors/test_checkpoint_connector.py

```diff
@@ -123,22 +123,22 @@ def test_loops_restore(tmpdir):
     trainer = Trainer(**trainer_args)
     trainer.strategy.connect(model)
 
-    for fn in TrainerFn:
-        if fn != TrainerFn.TUNING:
-            trainer_fn = getattr(trainer, f"{fn}_loop")
-            trainer_fn.load_state_dict = mock.Mock()
-
-    for fn in TrainerFn:
-        if fn != TrainerFn.TUNING:
-            trainer.state.fn = fn
-            trainer._checkpoint_connector.resume_start(ckpt_path)
-            trainer._checkpoint_connector.restore_loops()
-
-            trainer_loop = getattr(trainer, f"{fn}_loop")
-            trainer_loop.load_state_dict.assert_called()
-            trainer_loop.load_state_dict.reset_mock()
-
-        for fn2 in TrainerFn:
-            if fn2 not in (fn, TrainerFn.TUNING):
+    trainer_fns = [fn for fn in TrainerFn if fn != "tune"]
```

Comment:
Can you introduce something like `TrainerFn._without_tune()` that returns this list? This way we remember when we end up removing it

Reply:
done!